ui/package.json

```diff
@@ -77,7 +78,6 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "jest": "^29.6.1",
     "package-changed": "^3.0.0",
-    "playwright": "^1.36.0",
```

Comment:
Im probably wrong, but I thought I needed this for `mage dagger:run generate:screenshots`.
Just mentioning incase.